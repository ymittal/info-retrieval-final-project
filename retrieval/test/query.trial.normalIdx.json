{
    "casefold": true,
    "verbose": true,
    "index": ["../../idx/tedEd_en-ALL.idx"],
    "requested": 5,
    "queries": [{
        "number": "inTEDed",
        "text": "Sequestration",
    }, {
        "number": "inTEDed2",
        "text": "Morelli",
    }]
}
