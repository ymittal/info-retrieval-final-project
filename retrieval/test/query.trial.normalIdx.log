Dec 20, 2017 4:52:16 PM org.lemurproject.galago.core.tools.apps.BatchSearch run
INFO: RUNNING: inTEDed : sequestration
Dec 20, 2017 4:52:16 PM org.lemurproject.galago.core.tools.apps.BatchSearch run
INFO: Transformed Query:
#combine:w=1.0(
    #dirichlet:collectionLength=731787:maximumCount=2:nodeFrequency=2:w=1.0(
        #lengths:document:part=lengths()
        #counts:sequestration:part=postings.porter()
      )
  )

Dec 20, 2017 4:52:16 PM org.lemurproject.galago.core.tools.apps.BatchSearch run
INFO: RUNNING: inTEDed2 : morelli
Dec 20, 2017 4:52:16 PM org.lemurproject.galago.core.tools.apps.BatchSearch run
INFO: Transformed Query:
#combine:w=1.0(
    #dirichlet:collectionLength=731787:maximumCount=4:nodeFrequency=8:w=1.0(
        #lengths:document:part=lengths()
        #counts:morelli:part=postings.porter()
      )
  )
