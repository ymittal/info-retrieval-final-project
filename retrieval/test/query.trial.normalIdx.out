inTEDed Q0 rO6S93FKIUM 1 -7.14077972 galago
inTEDed2 Q0 B5x5CnwQZD4 1 -6.34810259 galago
inTEDed2 Q0 tVdw60eCnJI 2 -6.40113728 galago
